.github/CODEOWNERS

```
@@ -4,3 +4,12 @@ CODEOWNERS @DilumAluthge
 
 .ci/parse_env.jl @ericphanson
 .github/workflows/feed.yml @DilumAluthge @ericphanson
+
+C/Clang_assert_jll/ @vchuravy @staticfloat
+C/Clang_jll/ @vchuravy @staticfloat
+L/LLVMExtra_jll/ @vchuravy @staticfloat
```

```suggestion
L/LLVMExtra_jll/ @vchuravy @maleadt 
```

Don't know if @maleadt has the merge bit here.

---

I just checked, and he does.